HTML docs and even man pages require some extra Sphinx extensions
to be installed.

diff -upr kitty-0.24.3.orig/setup.py kitty-0.24.3/setup.py
--- kitty-0.24.3.orig/setup.py	2022-03-17 17:56:41.669108248 +0100
+++ kitty-0.24.3/setup.py	2022-03-17 17:57:47.155989536 +0100
@@ -943,11 +943,6 @@ def compile_python(base_path: str) -> No
 
 
 def create_linux_bundle_gunk(ddir: str, libdir_name: str) -> None:
-    if not os.path.exists('docs/_build/html'):
-        make = 'gmake' if is_freebsd else 'make'
-        run_tool([make, 'docs'])
-    copy_man_pages(ddir)
-    copy_html_docs(ddir)
     for (icdir, ext) in {'256x256': 'png', 'scalable': 'svg'}.items():
         icdir = os.path.join(ddir, 'share', 'icons', 'hicolor', icdir, 'apps')
         safe_makedirs(icdir)
